src/Pyz/Yves/CompanyPage/Theme/default/templates/page-layout-company/page-layout-company.twig

```diff
@@ -43,6 +43,12 @@
         {% block inner %}
             <div class="grid grid--gap">
                 <div class="col col--sm-12 col--xl-3 is-hidden-sm-lg">
+                    {% include atom('skip-link', '@SprykerShop:ShopUi') with {
+                        data: {
+                            href: '#company-content',
+                            text: 'global.skip-to-company-content.link.text' | trans,
```

Comment:
Do we need a specific text for it?

Reply:
Replaced with more general text version to use on both - company and customer pages